.buildkite/pipeline.yml

```diff
@@ -160,6 +160,18 @@ steps:
       provider: "gcp"
       machineType: "n1-standard-8"
 
+  - label: "Serverless Beats Tests"
+    key: "serverless-beats-integration-tests"
+    command: ".buildkite/scripts/steps/beats_tests.sh"
+    if: "build.env('CRON') == 'yes'"
```

Comment:
Where is the cron schedule controlled? Can you link to it here?

Reply:
So, I was going to add it once the PR was merged, but the control panel for it is here: https://buildkite.com/elastic/elastic-agent/settings/schedules

Reply:
Add a link to that as a comment right above this so future us can find it easily